src/components/nav-bar/navlink.tsx

```diff
@@ -0,0 +1,44 @@
+import { Block } from 'baseui/block';
+import { StyledLink } from 'baseui/link';
+import { Link } from 'gatsby';
+import React from 'react';
+
+type NavLinkProps = {
+  navigateTo: string;
+  text: string;
+  textColor: string;
+};
+
+const NavLink: React.FC<NavLinkProps> = ({ navigateTo, text, textColor }) => {
+  return (
+    <Block
+      overrides={{
+        Block: {
+          style: ({ $theme }) => ({
+            fontWeight: 400,
+            fontSize: $theme.sizing.scale550,
+          }),
+        },
+      }}
+    >
+      <Link to={navigateTo} style={{ textDecoration: 'none' }}>
+        <StyledLink
+          $style={({ $theme }) => ({
+            color: `${textColor} !important`,
+            cursor: 'pointer',
+            fontWeight: 400,
```

Comment:
This is not desirable to override fontweight here.

Can you please look into how baseweb recommends to style the links at a global level?